libs/txnbuild/price_update.go

```diff
@@ -37,6 +39,7 @@ func (priceUpdate *PriceUpdate) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for PriceUpdate initialises the vtxnbuild struct from the corresponding xdr Operation.
```

Comment:
FromXDR for PriceUpdate initialises the vtxnbuild struct from the corresponding **XDR** Operation.